### ISSUE TITLE
Offer `system`-style `pre-commit` hook alongside the `docker` one

### ISSUE THREAD

**Issue by Alexander-Shukaev** [koalaman/shellcheck#2946]
Is it possible to merge `system`-style hook (see https://github.com/Jarmos-san/shellcheck-precommit) to the official one please?  Thanks.

**Comment by skwde** [comment]
FYI, there is also <https://github.com/shellcheck-py/shellcheck-py>

**Comment by davidxia** [comment]
related to https://github.com/koalaman/shellcheck/issues/2495

**Comment by matthewsht** [comment]
Agreed - I just attempted this on macOS and while the official shellcheck install instructions suggest brew, the pre-commit wants docker -
